department-of-veterans-affairs/vets-website
# Add pension email and mailing address

## Summary
On the Pension Benefits application(21P-527EZ), moves the email/phone and mailing address sections to separate pages in Chapter 1 - Applicant Information, and updates the included fields for the 'Email address and phone numbers' page to match current designs. 

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Select 'Start your application without signing in'
3. Verify the page 'Mailing address' is in the correct location, per the designs, and includes only one 'Mailing address' title
4. Verify the page 'Mailing address' has a checkbox for "I receive mail outside of the United States on a U.S. military base" and has the dropdown for "Learn more about military base addresses"
5. Verify that when the checkbox for "I receive mail outside of the United States on a U.S. military base" is selected, the postal code must be a five-digit number.
6. Verify the page 'Email address and phone numbers' includes the fields email, phone number, mobile number, and international number. Email and mobile number should be required.

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70330)
[Ticket in GitHub](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70329)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots

### Desktop
| Before | After (Mailing address) | After (Mailing address verification) | After (Phone and email) |  
| ------ | ------ | ------ | ------ |
|<img width="349" alt="before" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/e180135c-168f-4766-bdce-2de4508a21df"> | <img width="454" alt="After" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/edf3b34d-5a70-4a04-88bd-8eded40c260d"> | <img width="458" alt="Screenshot 2023-11-29 at 3 13 12 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/867b1c1c-b01e-4f2e-95c5-fc329b953ccf"> | <img width="521" alt="email-phone" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/b70b09f9-1674-4f53-9675-067b39470283"> |

## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70330)

### Quality Assurance & Testing

### Error Handling


### Authentication
